server/models/schema.sql

```diff
@@ -1735,16 +1735,17 @@ CREATE TABLE `unit` (
 ) ENGINE=InnoDB DEFAULT CHARACTER SET = utf8mb4 DEFAULT COLLATE = utf8mb4_unicode_ci;
 
 DROP TABLE IF EXISTS `user`;
-
 CREATE TABLE `user` (
   `id`            SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
   `username`      VARCHAR(80) NOT NULL,
   `password`      VARCHAR(100) NOT NULL,
   `display_name`  TEXT NOT NULL,
-  `email`         VARCHAR(100) DEFAULT NULL,
+  `email`         VARCHAR(99) DEFAULT NULL,
```

Comment:
Why this change to the length of the email field?

Reply:
Good question.  The new year's palm wine must not have worn off when I wrote this :smile: